ui/css/style.css

```diff
@@ -1775,3 +1775,47 @@ textarea::placeholder {
     position: absolute;
     top: 230px;
     right: 5%; } }
+
+.profiles-container {
+  width: 100%; }
+
+.profile-card {
+  width: 100%;
+  display: flex;
+  height: 20vh; }
+
+.profile-image {
+  width: 20%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-right: 10px; }
+
+.profile-details {
+  width: 80%; }
+
+.profile-name {
+  height: 30%;
+  font-family: Roboto;
+  font-style: normal;
+  font-weight: bold;
+  line-height: normal;
+  font-size: 20px; }
+
+.profile-biography {
+  height: 70%;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  line-height: 2.8vh; }
+
+.follow-button {
+  width: 30%;
+  height: 90%;
+  margin-left: auto; }
+
+.follow-button button {
```

Comment:
@filschristian Try to use sass nested principle effectively to avoid overwriting other buttons

Reply:
Good feedback implemented, Thanks @oesukam